projects/storefrontapp/src/app/spartacus/features/digital-payments-feature.module.ts

```diff
@@ -17,4 +16,40 @@ import {
     }),
   ],
 })
-export class DigitalPaymentsFeatureModule {}
+export class DigitalPaymentsFeatureModule {
+  static forB2C(): ModuleWithProviders<DigitalPaymentsFeatureModule> {
+    return {
+      ngModule: DigitalPaymentsFeatureModule,
+      providers: [
+        provideConfig({
+          featureModules: {
+            [CHECKOUT_FEATURE]: {
+              module: () =>
+                import('@spartacus/digital-payments').then(
+                  (m) => m.DigitalPaymentsModule
+                ),
+            },
+          },
+        }),
+      ],
+    };
+  }
+
+  static forB2B(): ModuleWithProviders<DigitalPaymentsFeatureModule> {
```

Comment:
for b2b should point to b2b checkout

and another one for replenishment

`forScheduledReplenishment()` pointing to the replenishment/digitalpayment module